### PR TITLE
Free collaboration

### PR DESCRIPTION
**Pull request by ArifPMan**

### CODE REVIEW

File: v6/postman/launching_postman/collaboration.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+title: "Variables"
+page_id: "variables"
+warning: false
+
+---
+
+## **Collaboration Now Available for Free Users**
+
+With a view to encouraging real-time collaboration between developers and other stakeholders during any point in the API lifecycle, Postman now allows all free users to collaborate with their teams through Team Workspaces. Using this feature, you can easily collaborate and share your Collections, Environments, Integrations, History, Mocks, Monitors, and more. Keeping in line with its legacy of providing rich user experience, we, at Postman, have made collaboration as easy as sharing a bookmark.
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: Let's start with `Postman allows all free users to collaborate with their teams through Team Workspaces`

Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: Let's remove `Keeping in line with its legacy of providing rich user experience, we, at Postman, have made collaboration as easy as sharing a bookmark.`

File: v6/postman/launching_postman/collaboration.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+page_id: "variables"
+warning: false
+
+---
+
+## **Collaboration Now Available for Free Users**
+
+With a view to encouraging real-time collaboration between developers and other stakeholders during any point in the API lifecycle, Postman now allows all free users to collaborate with their teams through Team Workspaces. Using this feature, you can easily collaborate and share your Collections, Environments, Integrations, History, Mocks, Monitors, and more. Keeping in line with its legacy of providing rich user experience, we, at Postman, have made collaboration as easy as sharing a bookmark. 
+
+**Note:** If you are a Free User, you must download the latest Postman app version 6.2 to collaborate with other teams. More importantly, Postman offers an upper limit on the number of collection requests and history shares. For more information on the usage limit, refer to the section Usage Limit for Free Users. However, note that Postman does not impose any limit on the number of people you can invite.
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: Free collaboration is available only from Postman v6.2 onwards.

File: v6/postman/launching_postman/collaboration.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+**Note:** If you are a Free User, you must download the latest Postman app version 6.2 to collaborate with other teams. More importantly, Postman offers an upper limit on the number of collection requests and history shares. For more information on the usage limit, refer to the section Usage Limit for Free Users. However, note that Postman does not impose any limit on the number of people you can invite. 
+
+You can now get started creating team workspaces and inviting users to join your workspace just by sending an invitation through ‘invite’ button. You would also be able to join workspaces created by other team members for collaboration. This section explains how to collaborate in Postman and derive the many benefits it offers. It explains the following topics:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: `Invite

File: v6/postman/launching_postman/collaboration.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+This section explains different ways of creating a team workspace and inviting other team members for collaboration. You can create a team workspace using the following ways:
+
+* Inviting a team member to a personal workspace
+
+* Creating a new workspace from the menu
+
+#### **Inviting a Team Member to a Personal Workspace**
+
+You can use the invite button displayed prominently adjacent to the Workspace Switcher  . Click Invite. The following screen opens:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: Let's add a pic here highlighting the Invite button

File: v6/postman/launching_postman/collaboration.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+Click Create New in the above screen. The following screen opens:
+
+[![create workspace](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/CreateNewWorkspace.png)
+ 
+Specify a name in the Name field. Input any details for Summary and select Team under Type. The Team option appears highlighted in orange color indicating its selection. Note that the Invite members to this workspace option is available only for Team workspaces. 
+
+### **Usage Limit for Free Users**
+
+Traditionally, team workspace was available only for Postman Pro and Postman Enterprise users until Postman 6.2 version. However, conceding to the overwhelming demand in various fora like Postman Community and others, Postman opened the floodgates for all the users but with some controls. This section explains the following topics:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: Until Postman 6.2, Team workspaces were available only for Postman Pro and Postman Enterprise users. From Postman 6.2 onwards, Team workspaces are available to Free users as well with the following limits.

File: v6/postman/launching_postman/collaboration.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+* How Archiving Works
+
+* Recovering Your Archived Collections
+
+#### **Usage Limit**
+
+Postman allows you to collaborate with other team members on up to 25 shared requests and an equal number of shared history under the free plan. Once you exceed your limit, Postman archives your collections and shared history requests. However, Postman secures your archived collections and provides access to your data with options of exporting and downloading. Also, note that the usage limit does not apply to sharing environments; it applies only to shared collection requests and history. Only the collections that you share with other teams count towards this limit and you can still continue having unlimited collections for personal use. 
+
+**Note:** Postman archives only collections, it does not archive standalone requests.
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: Let's remove this.

File: v6/postman/launching_postman/collaboration.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+* Recovering Your Archived Collections
+
+#### **Usage Limit**
+
+Postman allows you to collaborate with other team members on up to 25 shared requests and an equal number of shared history under the free plan. Once you exceed your limit, Postman archives your collections and shared history requests. However, Postman secures your archived collections and provides access to your data with options of exporting and downloading. Also, note that the usage limit does not apply to sharing environments; it applies only to shared collection requests and history. Only the collections that you share with other teams count towards this limit and you can still continue having unlimited collections for personal use. 
+
+**Note:** Postman archives only collections, it does not archive standalone requests. 
+
+You can check the number of requests you shared using some of the features in Postman like Usage drop-down and through the Dashboard. This section illustrates this procedure. Click the arrow next to upgrade   button on top right corner of your Postman main screen. The following screen appears:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: > Usage drop-down

let's show the Upgrade button and a part of the header also in the figure below.

File: v6/postman/launching_postman/collaboration.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+[![usage info](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/UsageInfo1.png)
+ 
+The above screen provides you all the information related to your usage such as shared requests, history, mock usage and so on. 
+
+#### **How Archiving Works**
+
+Postman is always sensitive to the user data and it does a fairly competent job in archiving your collections and requests based on your usage. The reason why Postman archives your collections after you reach your usage limit is to still enable you to share your important collections without bringing your work to a complete standstill. Postman notifies you from time to time as your collections get archived, something like this:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: `Postman archives your collections after you reach your usage limit to keep enabling you to share your important collections without bringing your work to a complete standstill. Postman notifies you from time to time as your collections get archived, something like this:

File: v6/postman/launching_postman/collaboration.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+If you are still over the limit, Postman then analyzes the next oldest collection and archives Github-Support-Issues-Alert because it is the next oldest (edited about, let's say, three days ago). In an attempt to allow you to continuously share your work, Postman keeps a tab on the share limit to ensure your work is uninterrupted. Postman indicates the number of your archived collections at the bottom of the sidebar as illustrated in the screen below: 
+
+[![archived message](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/ArchiveMsg2.png) 
+
+The above screen displays six archived collections. 
+
+#### **Recovering Your Archived Collections**
+
+As Postman archives your collections, you might want to recover your collections for your reference and use. Postman does a remarkable job securing all of your data and enables you to easily access and retrieve the data. To recover your collections, in the Postman main screen, click archived collections, as illustrated below:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: let's remove "Postman does a remarkable job securing all of your data and enables you to easily access and retrieve the data"

Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: Let's capitalize "archived collections"

**Comment by ArifPMan** [comment]
Fixed all the comments....Please review and merge....

File: v6/postman/launching_postman/collaboration.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+* Recovering Your Archived Collections
+
+#### **Usage Limit**
+
+Postman allows you to collaborate with other team members on up to 25 shared requests and an equal number of shared history under the free plan. Once you exceed your limit, Postman archives your collections and shared history requests. However, Postman secures your archived collections and provides access to your data with options of exporting and downloading. Also, note that the usage limit does not apply to sharing environments; it applies only to shared collection requests and history. Only the collections that you share with other teams count towards this limit and you can still continue having unlimited collections for personal use. 
+
+You can check the number of requests you shared using some of the features in Postman like usage drop-down and through the Dashboard. The usage drop-down is accessible through the upgrade button on top right corner. It is shown below:
+
+[![usage drop-down](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Usage+drop-down.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Usage+drop-down.png)
```
Comment (numaanashraf) [inline v6/postman/launching_postman/collaboration.md]: The usage dropdown is opened by clicking the down arrow beside the Upgrade button. Not the Upgrade button. Highlight needs to be modified here.

**Pull request merged**